### PR TITLE
MO-3718: attempt to fix helm chart for serviceaccount

### PR DESCRIPTION
**Pull request by federico-nervegna-kayo**
Fix wrong indent in serviceaccount.yaml
Bump aws sdk to version support OIDC in EKS per https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts-minimum-sdk.html

### CODE REVIEW

**Review by hoaphanFox — APPROVED** [review]

**Comment by service-user-kayosportsau** [comment]
:star: PR built and available in a preview environment **fsa-streamotion-conductor-pr-101**

File: charts/netflix-conductor/templates/serviceaccount.yaml [hunk 1]
```diff
@@ -5,10 +5,10 @@ metadata:
...
-    app: {{ template "fullname" . }}
-    application: {{ template "fullname" . }}
+    app: {{ include "netflix-conductor.name" . }}-server
+    application: {{ include "netflix-conductor.name" . }}-server
   name: {{ .Values.serviceAccount.name }}
   {{- if .Values.serviceAccount.annotations }}
-annotations: {{ toYaml .Values.serviceAccount.annotations | nindent 4 }}
+  annotations: {{ toYaml .Values.serviceAccount.annotations | nindent 4 }}
   {{- end }}
   {{- end }}
```
Comment (hoaphanFox) [inline charts/netflix-conductor/templates/serviceaccount.yaml:14]: << I think it still works fine but it should be to the top level on the right, matching the <if> opening

**Comment by service-user-kayosportsau** [comment]
:star: PR built and available in a preview environment **fsa-streamotion-conductor-pr-101**

File: charts/netflix-conductor/templates/serviceaccount.yaml (hunk 1 above)
Comment (federico-nervegna-kayo) [reply charts/netflix-conductor/templates/serviceaccount.yaml:14]: just following what was done in bedrock: https://github.com/fsa-streamotion/foxsports-platform-bedrock-helm/blob/master/charts/helm-release/templates/serviceaccount.yaml

You know, consistency :)

**Review by hoaphanFox — APPROVED** [review]

**Review by benjamin-tucker — APPROVED** [review]

**Comment by service-user-kayosportsau** [comment]
:star: PR built and available in a preview environment **fsa-streamotion-conductor-pr-101**

**Comment by service-user-kayosportsau** [comment]
:star: PR built and available in a preview environment **fsa-streamotion-conductor-pr-101**

**Pull request merged**